src/content/en/updates/posts/2016/03/web-enabled-internet-of-things.markdown

```diff
@@ -0,0 +1,523 @@
+---
+layout: updates/post
+title: "Creating a Web-enabled IoT device with Intel&reg; Edison"
+description: "Internet of Things is really on everyone's lips these
+days, but what if you could create an IoT device which you accessed
+from your web browser? In this article we look at how you can do
+exactly that using Physical Web, Web Bluetooth as well as Node.JS,
+running on an Intel&reg; Edison Arduino breakout board."
+published_on: 2016-03-20
+updated_on: 2016-03-20
+authors:
+  - kenchris
+tags:
+  - css
```

Comment:
You might want to set some tags there:
  - news
  - Bluetooth
  - IoT
  - WebBluetooth
  - PhysicalWeb
  - Edison